app/src/main/res/layout/feedback_item.xml

```diff
@@ -48,7 +48,7 @@
       android:breakStrategy="simple"
       android:fontFamily="sans-serif"
       android:text="@{htmlContent}"
-      android:textColor="@color/oppia_primary_text"
+      android:textColor="@color/component_color_question_player_activity_text_color"
```

Comment:
This is incorrect because this item is getting used in `exploration_player` too. Therefore I suggest that you renaming it to `component_color_shared_feedback_item_text_view_color`

Reply:
Done